clientlibrary/checkpoint/dynamodb-leasestealer_test.go

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (c) 2018 VMware, Inc.
```

Comment:
Nit. 2020

Reply:
This file no longer exists. Feel free to resolve.